ultra/docs/getting_started.md

```diff
@@ -0,0 +1,112 @@
+# Getting Started
+
+The following shows how to generate left-turn scenarios for a specific task, then train an agent on those scenarios, and finally evaluate the agent. We will use Task 1 as an example.
+
+## Generating Scenarios for a Task
+
+To begin, start by generating scenarios for a left-turn task. Each task has various left-turn scenarios that aim to teach the agent something. Task 1 trains the agent to generalize its knowledge on the type of intersection. This is done by creating left-turn scenarios in t-intersections for training, and left-turn scenarios in cross-intersections for testing. All supported tasks are listed under `ultra/scenarios/`.
```

Comment:
I believe adding an graphic to visualize the different tasks (task1 and task2) can provide a better visualization of left turn problem to some people

Reply:
Oh yeah sounds good. I will add graphics for just Task 1 for now as Task 2 may be harder to show differences in traffic density.